EasyPost.Net/Client.cs

```diff
@@ -129,7 +139,6 @@ private RestRequest PrepareRequest(Request request)
         {
             RestRequest restRequest = (RestRequest)request;
             restRequest.Timeout = RequestTimeoutMilliseconds;
-            restRequest.AddHeader("user_agent", UserAgent);
```

Comment:
Is this being done somewhere else?

Reply:
Yes, the `UserAgent` is added as a property of the RestSharp client during initialization.